cmd/chasquid-util/chasquid-util.go

```diff
@@ -65,6 +66,7 @@ func main() {
 		"aliases-resolve":   aliasesResolve,
 		"print-config":      printConfig,
 		"domaininfo-remove": domaininfoRemove,
+		"alias-add":         aliasAdd,
```

Comment:
Move this to below aliases-resolve, and rename aliases-add  for consistency?

Reply:
I wanted it to be consistent with `user-add` but I can also rename it if you prefer that

Reply:
If you don't mind, yeah, I'd prefer it.

At some point it might be better to rename "aliases" with "alias" but I rather leave that for a separate series.

Reply:
Done :)